File: FirstActivityProject/app/src/main/java/net/daan/firstactivityproject/MainActivity.java
package net.daan.firstactivityproject;

import android.support.annotation.NonNull;
import android.support.design.widget.BottomNavigationView;
import android.support.v4.app.Fragment;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.MenuItem;

public class MainActivity extends AppCompatActivity {

    private BottomNavigationView bottomNavigationView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        bottomNavigationView = findViewById(R.id.bottom_navigation_view);
        bottomNavigationView.setOnNavigationItemSelectedListener(navListener);

        getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container, new LoginFragment()).commit();
    }

    private BottomNavigationView.OnNavigationItemSelectedListener navListener = new BottomNavigationView.OnNavigationItemSelectedListener() {
        @Override
        public boolean onNavigationItemSelected(@NonNull MenuItem item) {

            Fragment fragment = null;

            switch (item.getItemId()) {
                case R.id.nav_login:
                    fragment = new LoginFragment(); break;

                case R.id.nav_faq:
                    fragment = new FaqFragment(); break;

                case R.id.nav_termos:
                    fragment = new TermosFragment(); break;
            }

            getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container, fragment).commit();

            return true;
        }
    };
}

File: RecyclerView/app/src/main/java/net/daan/recyclerview/MainActivity.kt
package net.daan.recyclerview

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import kotlinx.android.synthetic.main.activity_main.*
import net.daan.recyclerview.adapter.DentistaRecyclerAdapter

class MainActivity : AppCompatActivity() {

    private lateinit var dentistaRecyclerAdapter: RecyclerView.Adapter<RecyclerView.ViewHolder>

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        init()
    }

    private fun init() {
        recyclerViewDentistas.layoutManager = LinearLayoutManager(this)
        dentistaRecyclerAdapter = DentistaRecyclerAdapter(DataSource.createDataset())
        recyclerViewDentistas.adapter = dentistaRecyclerAdapter
    }
}
